Atomic/scan.py

```diff
@@ -517,3 +533,7 @@ def write_persistent_data(self):
 
         with open(summary_file, 'w') as f:
             json.dump(persistent_data, f, indent=4)
+
+
+    def remediate(self, script, iid, results_dir):
+        util.check_call([sys.executable, script, '--id {}'.format(iid), '--results_dir {}'.format(results_dir)])
```

Comment:
Shouldn't these be all separate.

   util.check_call([sys.executable, script, '--id', '{}'.format(iid), '--results_dir', '{}'.format(results_dir)])

Reply:
I think I will separate them but not use the format statements, they are no useful here.